SteakBot.Core/EventHandlers/VoiceStateEventHandler.cs

```diff
@@ -1,33 +1,64 @@
-﻿using System.Linq;
-using System.Threading.Tasks;
-using Discord.WebSocket;
+﻿using Discord.WebSocket;
 using SteakBot.Core.EventHandlers.Abstraction;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
```

Comment:
Since I couldn't find a proper list here's a breakdown. Using directives are broken into groups that determine their order:
 - System namespaces
 - Project namespaces
 - External/Third-party namespaces
 - Aliases

They should appear in that order. Each group is internally sorted alphabetically.
Reference StyleCop rules - http://stylecop.soyuz5.com/SA1208.html through http://stylecop.soyuz5.com/SA1211.html